README.md

```diff
@@ -15,19 +15,48 @@ This [awesome](https://github.com/sindresorhus/awesome) repository is maintained
 
 - [Books](#books)
 - [Blogs](#blogs)
-- [Articles](#articles)
-- [Videos](#videos)
-- [Repositories](#repositories)
+- [Tools and Applications](#tools-and-applications)
+- [Articles and Videos](#articles-and-videos)
 - [Adopters](#adopters)
 
 ## Books
 
+- [Getting Started With InnerSource](https://innersourcecommons.org/learn/books/getting-started-with-innersource/) - Author Andy Oram takes you inside InnerSource, first by reviewing the principles that make open source development successful, and then by describing how InnerSource has worked at PayPal.
+- [Understanding the InnerSource Checklist](https://innersourcecommons.org/learn/books/understanding-the-innersource-checklist/) - A checklist guides you through various steps needed to produce software using the InnerSource approach, whether it’s company wide or just for one team.
+- [Managing InnerSource Projects](https://innersourcecommons.org/learn/books/managing-innersource-projects/) - In this book Daniel Izquierdo & José Manrique López explain basic infrastructure as well as metrics that are helpful when introducing InnerSource methodologies into an enterprise.
+- [Adopting InnerSource](https://innersourcecommons.org/learn/books/adopting-innersource-principles-and-case-studies/) - The case studies candidly discuss the difficulties of getting InnerSource projects started, along with the progress so far and the benefits or negative fallout.
+- [InnerSource Patterns](https://innersourcecommons.org/learn/books/innersource-patterns/) -  This book contains the most mature patterns, collected by the InnerSource Commons Community.
+
 ## Blogs
 
-## Articles 
+- [Sebastian Spier - Blog](https://spier.hu/) - Sebastian Spier is the Board member of the [InnerSource Commons Foundation](https://innersourcecommons.org/).
+
+## Tools and Applications
+
+- [Backstage](https://backstage.io/) - An open platform for building developer portals. Realization of the [InnerSource Portal Pattern](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/innersource-portal.md).
```

Comment:
You will find more reference implementations of the InnerSource Portal pattern in here from SAP and Elbit.
https://patterns.innersourcecommons.org/p/innersource-portal#references

Begs the question though whether we should repeat that content from the pattern in here?

Reply:
I think we should mention them